src/compiler/parser.ts

```diff
@@ -611,44 +611,24 @@ namespace ts {
                 fixupParentReferences(sourceFile);

```

Comment:
There are several calls to 'fixupParentReferences` from within the JSDoc parser, but only the above call is necessary because JSDoc comments are no longer parsed after the main pass.